### PR TITLE
Adds component library as a dependency

### PR DESCRIPTION
**Pull request by DingoEatingFuzz**
Now that the component library has an [official home](https://www.npmjs.com/package/@hackoregon/component-library), it can be add to this project.

~In doing this, I ran into trouble with dependencies and webpack configuration. My approach to fixing webpack configs is slamming my head against the keyboard until things build right, so there may be better solutions.~

Turns out you have to import components from `lib`, not `src` 😬

### CODE REVIEW

**Comment by meganmckissack** [comment]
This worked for me using the budget front-end repo, which is just a clone of this one.  The other teams' front-end repos will need this update too.  I can merge this pr if it's alright with you.

**Comment by DingoEatingFuzz** [comment]
I'm happy to hear it works, but I'm hoping to get a review from @DavideDaniel on the implementation. I'm still not sure if I did it the ideal way. Particularly the changes to the webpack config files.

**Review by dsdemaria — APPROVED** [review]

**Pull request merged**
